src/main/java/org/broadinstitute/consent/http/resources/DaaResource.java

```diff
@@ -115,19 +115,18 @@ public Response createLibraryCardDaaRelation(
           return Response.status(Status.FORBIDDEN).build();
       }
       List<LibraryCard> libraryCards = libraryCardService.findLibraryCardsByUserId(userId);
-      Optional<LibraryCard> matchingCard = libraryCards.stream()
-          .filter(card -> card.getInstitutionId() == authedUser.getInstitutionId().intValue())
-          .findFirst();
-      if (matchingCard.isEmpty()) {
-        LibraryCard createdLc = libraryCardService.createLibraryCardForSigningOfficial(user, authedUser);
-        matchingCard = Optional.of(createdLc);
+      LibraryCard workingLibraryCard;
+      if (libraryCards.isEmpty()) {
+        workingLibraryCard = libraryCardService.createLibraryCardForSigningOfficial(user, authedUser);
+      } else {
+        workingLibraryCard = libraryCards.get(0);
```

Comment:
A future PR will restrict users to having a single library card. In the meantime, this should preserve the existing behavior.

Reply:
Since we're using JDK 21 we can use `getFirst()` and `getLast()` which work on ordered collections.
```suggestion
        workingLibraryCard = libraryCards.getFirst();
```

Similar to `size() == 0` vs `isEmpty()` I think `getFirst()` is easier to read than `get(0)`.

Reply:
This doesn't compile for me locally even though it is JDK 21 - not sure why that is. I'd like to tackle that separately.